nucliadb/nucliadb/train/__init__.py

```diff
@@ -19,5 +19,20 @@
 
 import logging
 
-SERVICE_NAME = "nucliadb_train"
+SERVICE_NAME = "nucliadb.train"
+API_PREFIX = "api"
 logger = logging.getLogger(SERVICE_NAME)
+
+
+# Define the filter
+class EndpointFilter(logging.Filter):
+    def filter(self, record: logging.LogRecord) -> bool:
```

Comment:
Can you add a comment making clear what lines are you trying to filter out?

Reply:
This same code is on all components

Reply:
maybe we could do a PR to document all of them